explainer.md

```diff
@@ -275,16 +280,27 @@ class CustomCheckbox extends HTMLElement {
 customElements.define("x-checkbox", CustomCheckbox);
 ```
 
+From the perspective of the embedding page, a custom checkbox would now
+have this clean HTML, with the accessibility fully encapsulated into
+the implementation rather than exposed as a leaky abstraction:
+
 ```html
 <custom-checkbox checked>Receive promotional offers</custom-checkbox>
 ```
 
 Furthermore, writing Accessible Properties would allow specifying accessible relationships without requiring IDREFs,
-as authors can now pass DOM object references:
+as authors can now pass object references. Currently attributes like aria-describedby and aria-activedescendant
```

Comment:
Actually no - they're references to AccessibleNodes, and in later
phases they could be virtual AccessibleNodes that aren't related
to DOM objects. Right?

In other words, consider:

<input id="textbox">
<div id="label">First name</div>

The correct syntax is:

(1) $('textbox').accessibleNode.labeledBy = [$('label').accessibleNode]

not:

(2) $('textbox').accessibleNode.labeledBy = [$('label')]